astropy/coordinates/angles.py

```diff
@@ -209,6 +209,7 @@ def to_string(
         pad=False,
         fields=3,
         format=None,
+        oldstyle=False
```

Comment:
This name is not very informative. `Quantity.to_string`, which `Angle` as a `Quantity` subclass should ideally follow, has `subfmt`, which I guess one could use instead.